client/my-sites/site-settings/form-writing.jsx

```diff
@@ -163,10 +154,6 @@ const connectComponent = connect(
 		return {
 			siteIsJetpack,
 			siteId,
-			isMasterbarSectionVisible:
-				siteIsJetpack &&
-				// Masterbar can't be turned off on Atomic sites - don't show the toggle in that case
```

Comment:
I think it would be useful to keep that comment in the new place where we have the business logic for rendering the masterbar.

Reply:
Done!